src/assets/lang/locale-en.json

```
@@ -136,7 +136,7 @@
 		"ofLabel": "of"
 	},
 	"columnSettings": {
```

```suggestion
	"tableSettings": {
```

Da die Component jetzt auch Table anstatt Columns Settings heißt, bietet sich das sicherlich an. Entsprechend dann auch in den anderen Übersetzungen berücksichtigen und dort, wo die Keys genutzt werden